test/change.test.js

```diff
@@ -234,6 +287,25 @@ describe('Change', function() {
     });
   });
 
+  describe('change.currentRevision - promise variant', function() {
+    it('should get the correct revision', function(done) {
+      var test = this;
+      var change = new Change({
+        modelName: this.modelName,
+        modelId: this.modelId
+      });
+
+      change.currentRevision()
+      .then(function(rev) {
+        assert.equal(rev, test.revisionForModel);
+        done();
+      })
+      .catch(function(err) {
+        done(err);
+      });
```

Comment:
In this particular case, we cannot pass `done` as the second argument of `then`, because the error thrown by `assert.equal` would not be passed to that error handler.

So just simplify the `catch` block here by passing in `done` as the argument, no need to create a dummy argument-forwarding function.

```js
.catch(done);
```